plugins/woocommerce/tests/legacy/unit-tests/importer/product.php

```diff
@@ -363,8 +363,8 @@ public function test_get_parsed_data() {
 				'sale_price'            => '18',
 				'regular_price'         => '20',
 				'shipping_class_id'     => 0,
-				'download_limit'        => 0,
-				'download_expiry'       => 0,
+				'download_limit'        => '',
```

Comment:
For test, what do you think about taking it a step further? it would be great to ensure they not only check that the CSV parser returns '' but also verify the actual state of the product after import.

Reply:
Thanks for the comment! 

The test I updated is has a very specific context and is meant to test that the Importer parses data correctly, not that the data is actually set on the product. Once the data is parsed, then the Importer calls `set_props` to set them -- this function is tested by separate unit tests.